src/components/CustomSelect/CustomSelect.tsx

```diff
@@ -0,0 +1,58 @@
+import "./CustomSelect.scss"
+import {arrowSlider} from "../../assets/icons/arrowSlider"
+import { useState } from "react"
+
+export const CustomSelect = (props:{specialityList:Array<string>, userSpeciality:Array<string>, maxSelectedItem:number, callback:Function}) => {
```

Comment:
`specialityList:Array<string>` - better, if we use `string[]`, this is more commonly used and generally preferred for its brevity and readability, better with typical TypeScript conventions, but not necessary

Reply:
Ok, a fix this